src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,27 @@
 import React from 'react';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+import './CommentInfo.scss';
+
+import { Comment } from '../../types/comment';
+
+type Props = {
+  comment: Comment,
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
```

Comment:
It is recommended to use an interface instead of a type alias for component props to achieve better intellisense and code navigation experience.

Reply:
i used interface for start dates. I wrote these interfaces in folder types and they're imported in this type.